cmd/scripts/k8s/setup-host.sh

```diff
@@ -1,12 +1,15 @@
 #!/bin/bash
 
+# Install Cedana
+cp /usr/local/bin/cedana /host/usr/local/bin/cedana
+
```

Comment:
also copy the build-start-daemon from where it was extracted by the dockerfile

Reply:
just copy it to /host/build-start-daemon.sh ?

Reply:
yeah looks like that's where the setup script is running build-start-daemon from

Reply:
oh wait we still do a git clone nvm then